geography/index.html

```diff
@@ -165,7 +165,7 @@ <h5> Paleomagnetism.org 2 - Magnetostratigraphy </h5>
                <br>
                <div class="alert alert-light alert-dismissible fade show" role="alert">
                   <h5> Geography - Map Overview & APWPs </h5>
-                  <p> This map overview illustrates vertical-axis rotations for the selected sites including confidence parachutes (ΔDx). Declinations, inclinations, paleolatitudes, and apparant polar wander paths can be plotted for major plates. <p> <b>Tip!</b> You can create your own APWPs using <a href='https://apwp-online.org'>APWP-Online</a> by Vaes et al., 2023.
+                                   <p> This map overview illustrates vertical-axis rotations for the selected sites including confidence parachutes (ΔDx). Declinations, inclinations, paleolatitudes, and apparant polar wander paths can be plotted for major plates. <p> <b>Tip!</b> You can create your own APWPs using <a href='https://apwp-online.org'>APWP-Online</a> by Vaes et al., 2023.
```

Comment:
Same here

Reply:
Hi!
The previous path needed a rotation file to work because the plate circuit was incomplete. I have added Bram's full plate circuit at the end of the database (to keep the previous chronological order). The two changes you mention, I remember that there was some conflict between dev version and master versions (both have some commits ahead of the other) so I decided to use one of them (I guess the master?) and copy some stuff from one to other. Also I have corrected Vaes et al., 2022 for 2023 where it was 2022 (it was finally published in 2023). I have tested it and it works fine in my dummy version (dpastorgalan.github.io).

A thing that would be easier for next changes is if we make sure that the only one that is ahead is dev. (?)

Reply:
Always branch from dev and commit to dev and there should be no conflicts. Git can be a bit tricky though. Can you commit a change that fixes the problems I highligted? It changed the text from geography to statistics in the geography portal.

Reply:
I think I have correct this properly. Now it is geography again. I have also corrected the apparant for apparent typo. Since I had a mess, I have decided to use my own dev branch forked from yours to submit all other changes that I have in my playground branches. So I can go one by one and avoid messing around again. Cheers!